dynomitemanager/src/main/java/com/netflix/dynomitemanager/defaultimpl/DynomiteManagerConfiguration.java

```diff
@@ -146,21 +150,37 @@ private int getIntProperty(String envVar, String key, int defaultValue) {
 
     private static final String CONFIG_DYNOMITE_PROCESS_NAME = DYNOMITE_PROPS + ".process.name";
     private static final String CONFIG_DYNOMITE_YAML = DYNOMITE_PROPS + ".yaml";
-    private static final String CONFIG_SECURED_OPTION = DYNOMITEMANAGER_PRE + ".secured.option";
-    private static final String CONFIG_DYNO_AUTO_EJECT_HOSTS = DYNOMITEMANAGER_PRE + ".auto.eject.hosts";
+    private static final String CONFIG_DYNOMITE_INTRA_CLUSTER_SECURITY = DYNOMITE_PROPS + ".intra.cluster.security";
```

Comment:
@ipapapa I plan to commit a script to convert from the old names to the new names once the updates are done.

`./upgrade-properties /path/to/dynomitemanager.properties`

The update script combined with backward compatibility for env vars should make the change a 30 second operation for users.

IMO, the right time to clean up the property names is now as we are pre-Archaius in `master`. This means that DM users either don't use the `.properties` file yet, or if they do we can quickly update it for them with a script (as above). After Archaius is in use, then updating properties becomes extremely difficult as there's no automated way to update the various Archaius backends that can be used to provide properties.

My plan is as follows:
1. Finish implementing Archaius / cleaning up FP names
2. Write script to update old `dynomitemanager.properties` from the old FP names to the new FP names
3. Update the documentation to inform users about the upgrade script

I think this should make updating the properties easy/painless for users.

> The update script will basically be a hash of old_fp -> new_fp, then I'll just substitute the old FP name with the new name.

Let me know if this adequately addresses the update issue.